src/utility.js

```diff
@@ -71,6 +71,19 @@ function isFunction(f) {
   return isType(f, 'function');
 }
 
+/* isNativeFunction - a convenience function for checking if a value is a native JS function
+ *
+ * @param f - any value
+ * @returns true if f is a native JS function, otherwise false
+ */
+function isNativeFunction(f) {
+  if (!isFunction(f)) {
+    return false;
+  }
+  var functionString = Function.prototype.toString.call(f);
+  return /\{\s+\[native code\]/.test(functionString);
+}
```

Comment:
FWIW, `lodash` has an `isNative` function that might be more robust. It essentially does a check for `[native code]`, but it does it by actually getting the toString result of a native code function and doing a bunch of regex magic to turn it into a regex to match against other native functions, probably to make up for any minor differences between browsers.

https://github.com/lodash/lodash/blob/master/isNative.js